WebRoot/css/zm.css

```diff
@@ -2176,7 +2168,42 @@ div.CompDragArea * {
 }
 
 .ZmTaskEditView {
-	@PageBg@
+	@TaskEditView@
+}
+
+.ZmTaskEditView .ZmTaskEditHeader {
+	@TaskEditHeader@
+}
+
+.ZmTaskEditView .ZmTaskEditContent {
+	@TaskEditContent@
+}
+
+.ZmTaskEditHeader .ZmTaskEditHeaderTable {
+	@TaskEditHeaderTable@
+}
+
+.ZmTaskEditHeader .ZmTaskEditHeaderTable > tbody > tr > td {
+	@TaskEditHeaderTableCell@
+}
+
+.ZmTaskEditHeaderTable .ZmFieldsetContent td,
+.ZmTaskEditHeaderTable .ZmFieldsetContent div,
+.ZmTaskEditHeaderTable .ZmFieldsetContent label {
+	@TaskEditViewFontSize@
```

Comment:
can't we override the individual declarations that target these? (if possible, otherwise will have to go with this only)